pyMBE.py

```diff
@@ -60,7 +80,7 @@ def __init__(self, SEED, temperature=None, unit_length=None, unit_charge=None, K
         self.Kw=Kw*self.units.mol**2 / (self.units.l**2)
         self.units.define(f'reduced_energy = {self.kT}')
         self.units.define(f'reduced_length = {unit_length}')
-        self.units.define(f'reduced_charge = 1*e')
+        self.units.define('reduced_charge = 1*e')
```

Comment:
Hey! The reduced charge should be `f'reduced_charge = {unit_charge}'`, not a hardcoded unit of elementary charge. Right now, the `unit_charge` provided by the user is silently ignored. Also, this block of code that defines defaults is duplicated from `set_reduced_units()`, which we could call instead. Please check c7fff1658f2.

Reply:
@jngrad good catch!! thank you for fixing this